app/controllers/index.js

```diff
@@ -14,7 +14,10 @@ export default Controller.extend({
       const newFragment = get(this, 'extension').addFragment();
       if (!newFragment) return;
       const uuid = get(newFragment.findBy('key', 'uuid'), 'value');
-      set(this, 'editingUUID', uuid);
+      // TODO: re-enable this after muuri rerender on "iEditing"
+      // bug is fixed
+      // set(this, 'editingUUID', uuid);
```

Comment:
So was this the problem child? While editing it doesn't factor in new elements?

Reply:
This is the outstanding issue... the grid layout does not recalculate heights upon clicking "edit", and the method to force it to do so is not working.
<img width="400" alt="screen shot 2019-01-11 at 11 32 13 am" src="https://user-images.githubusercontent.com/7025087/51046632-958e8800-1594-11e9-92b3-364969fa292b.png">